apps/coscrad-frontend/src/components/higher-order-components/categorizable-page.tsx

```diff
@@ -50,16 +47,22 @@ export const CategorizablePage = <T extends CategorizableType>({
           )
         : AggregateDetailContainer;
 
-    const EnhancedDetailPresenterFactory = shouldShowCommands
-        ? (categorizableType: CategorizableType) =>
-              WithCommands(
-                  // @ts-expect-error FIX ME
-                  detailPresenterFactory(categorizableType),
-                  // @ts-expect-error FIX ME
-                  ({ actions }) => actions,
-                  (_) => compositeIdentifier
-              )
-        : detailPresenterFactory;
+    const EnhancedDetailPresenterFactory = (categorizableType: CategorizableType) => {
+        // @ts-expect-error FIX ME
+        const DetailPresenter = detailPresenterFactory(categorizableType);
+
+        return (viewModel) => (
```

Comment:
This is some kind of magic here.

Reply:
Yes, that's what scares me. I don't like being the only one who can tweak this, although, it's very rare to need to and it makes it so outside of here the game is just define the index and detail presenters.